client/dashboard/utils.js

```diff
@@ -57,15 +59,14 @@ export function getProductIdsForCart(
 	}
 
 	const productIds = [];
-	const plugins = getSetting( 'plugins', {} );
 	const productTypes = profileItems.product_types || [];
 
 	productTypes.forEach( ( productType ) => {
 		if (
 			onboarding.productTypes[ productType ] &&
 			onboarding.productTypes[ productType ].product &&
 			( includeInstalledItems ||
-				! plugins.installedPlugins.includes(
+				! installedPlugins.includes(
```

Comment:
I'm getting a `Cannot read property 'includes' of undefined` here :-(

Reply:
Oops, I forgot to pass in an argument `installedPlugins`. Fix in https://github.com/woocommerce/woocommerce-admin/pull/4362/commits/9e9d7f5e2a5c669a4517668da4f2317fd0cdec5f